jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -85,6 +86,15 @@ public static long getCheckpointRestoreNanoTimeDelta() {
 		return checkpointRestoreNanoTimeDelta;
 	}
 
+	/**
+	 * Get the start time of the CRIU restore process.
+	 *
+	 * @return the time in nanoseconds since the epoch, 0 if restore has not occurred.
```

Comment:
should be -1

Reply:
I will remove `getCRIURestoreStartTime` and `getCRIURestoreStartTimeImpl`, and return `criuRestoreStartTimeInNanoseconds` in `getLastRestoreTimeImpl`.

Reply:
Oh I see what you did, you added a separate native. Interesting, I think we will need the original one for debugging pruposes only. So this version can stay.

Reply:
but it should still return -1 if it hasnt been restored yet

Reply:
Since the original native `getLastRestoreTimeImpl` could be used for debugging purposes, I will keep the new native `getCRIURestoreStartTimeImpl` and it will return -1 if restore did not occur yet.